pkg/commands/test/output.go

```diff
@@ -2,6 +2,7 @@ package test
 
 import (
 	"github.com/logrusorgru/aurora"
+    "fmt"
```

Comment:
I think `gofmt` broke here

Reply:
Thanks, fixed. I'll hook up `go fmt` in the Circle build to catch similar things and do a cleanup.